using-mobile-extensions/mobile-core/platform-services.md

```diff
@@ -146,3 +148,128 @@ This step should occur prior to any other interactions with the AEP SDK. While i
     }
 ```
 
+## iOS
+
+### 1. Conform to the ACPHttpConnectionPerformer protocol
+
+The `ACPHttpConnectionPerformer` is a protocol which must be conformed to in order to override the network stack. It provides two methods which must be implemented:
+
+    1. `shouldOverride`, which takes a URL and an HTTP method. You return true if you want the network stack to be overriden     by the performer, or false if not. 
+    2. `requestUrl`, which is the URL request override with a completion block.
+    
+### 2. Create an ACPHttpConnection and pass relevant data from your NSURLSessionDataTask completion block
+
+The completion block for the `requestUrl` method, takes an `ACPHttpConnection` as its parameter. The `ACPHttpConnection` is used when overriding the network stack in place of the internal network connection implementation and represents the response to an HTTP request. It is to be created using the NSURLResponse, and NSData from your url request response. In the case of a Network Error, or timeout, the `ACPHttpConnection*` is expected to be nil.
```

Comment:
Take the comma out of this sentence: The completion block for the `requestUrl` method, takes an `ACPHttpConnection` as its parameter.

Reply:
Take the comma out of this sentence: It is to be created using the NSURLResponse, and NSData from your url request response.